sw/device/sca/sha3_serial.c

```diff
@@ -429,14 +355,53 @@ static void kmac_init(void) {
       .entropy_mode = kDifKmacEntropyModeSoftware,
       .entropy_seed = {0xaa25b4bf, 0x48ce8fff, 0x5a78282a, 0x48465647,
                        0x70410fef},
-      .entropy_fast_process = false,
-      .msg_mask = true,
+      .entropy_fast_process = true,
+      .msg_mask = false,
   };
+
   SS_CHECK_DIF_OK(dif_kmac_configure(&kmac, config));
 
   kmac_block_until_idle();
 }
 
+/**
+ * Reinitializes the KMAC peripheral with new mask values.
+ *
+ * This function reconfigures KMAC with the appropriate mask setting.
+ */
```

Comment:
The function name and description might be counter intuitive. IIRC, the entropy seed can only be set once after reset. After this point, you can just change the values for `entropy_mode`, `entropy_fast_process` and `msg_mask`. But I might be wrong. It's worth to check the spec and determine whether this function should be changed.